### PR TITLE
Fixes some grammar issues with blood filtration.

### LINKED ISSUE #57226: Blood filter surgery always uses "you're" even when you're not the one being operated on.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Bruising is spelled bruising. Also, threw on some \the defines in the middle of the proc to prevent "The the blood filter" from occuring during surgery.

## Why It's Good For The Game

Improves visual clarity. Fixes #57226.

## Changelog
:cl:
spellcheck: Blood Filtration wording is better & spelled correctly.
/:cl:

### CODE REVIEW

File: code/modules/surgery/blood_filter.dm [hunk 1]
```diff
@@ -24,21 +24,21 @@
 
 /datum/surgery_step/filter_blood/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	display_results(user, target, "<span class='notice'>You begin filtering [target]'s blood...</span>",
-		"<span class='notice'>[user] uses the [tool] to filtering your blood.</span>",
-		"<span class='notice'>[user] uses the [tool] on [target]'s chest.</span>")
+		"<span class='notice'>[user] uses \the [tool] to filtering your blood.</span>",
+		"<span class='notice'>[user] uses \the [tool] on [target]'s chest.</span>")
```
Comment (Mothblocks) [inline code/modules/surgery/blood_filter.dm:28]:
```suggestion
		"<span class='notice'>[user] uses [tool] to filtering your blood.</span>",
		"<span class='notice'>[user] uses [tool] on [target]'s chest.</span>")
```

`\the` is automatic.

Comment (ArcaneMusic) [reply code/modules/surgery/blood_filter.dm:28]: Is that the case for all \the 's seen above and below, or just those specifically?

Comment (Mothblocks) [reply code/modules/surgery/blood_filter.dm:28]: `\The` isn't implicit, but `\the` is.

File: code/modules/surgery/blood_filter.dm [hunk 2]
```diff
@@ -24,21 +24,21 @@
 
 /datum/surgery_step/filter_blood/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	display_results(user, target, "<span class='notice'>You begin filtering [target]'s blood...</span>",
-		"<span class='notice'>[user] uses the [tool] to filtering your blood.</span>",
-		"<span class='notice'>[user] uses the [tool] on [target]'s chest.</span>")
+		"<span class='notice'>[user] uses [tool] to filtering your blood.</span>",
```
Comment (Mothblocks) [inline code/modules/surgery/blood_filter.dm:27]:
```suggestion
		"<span class='notice'>[user] uses [tool] to filter your blood.</span>",
```

**Pull request merged**